### ISSUE TITLE
🚀 Enhancement: Attribute Slider!

### ISSUE THREAD

**Issue by portmappings** [appwrite/console#1615]
### 🔖 Enhancement description

You should add a slider for organizing the list of attributes in collections.

Example from Clickup
![Image](https://github.com/user-attachments/assets/46727e90-4884-4636-bca4-ff730d4ca79c)
![Image](https://github.com/user-attachments/assets/64c27a20-eb9b-498d-951f-1a44e81c39be)
![Image](https://github.com/user-attachments/assets/bdd176a8-2491-48c3-be70-b799bad1550b)

### 🎤 Pitch

This would be useful for people to organize their data schemas, after deleting and creating new attributes with time, they get ugly and disorganized.

### 👀 Have you spent some time to check if this issue has been raised before?

- [x] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [x] I have read the [Code of Conduct](https://github.com/appwrite/.github/blob/main/CODE_OF_CONDUCT.md)

**Comment by FrankVanris2** [comment]
I'll work on this

**Comment by FrankVanris2** [comment]
@portmappings Yo quick question, when you say slider, do you mean a slider icon that moves the screen from top to bottom or left and right? or do you mean a drag and drop feature for the different data schemas? Cause when I think of sliders it usually means an icon that allows you to move from top/bottom or left/right of a screen.

**Comment by portmappings** [comment]
Yes, a drag and drop to organize the attributes. I was trying to find the name for that.

**Comment by FrankVanris2** [comment]
Ok lol, that is good to know

**Comment by eldadfux** [comment]
Starting today this repo is archived. Please raise any issues with the new Appwrite console on the main platform repo: https://github.com/appwrite/appwrite
